_assets/ci/Jenkinsfile

```diff
@@ -3,7 +3,7 @@ pipeline {
 
   parameters {
     booleanParam(
-      name: 'GITHUB_RELEASE',
+      name: 'RELEASE',
       description: 'Enable this to create a new GitHub release.',
```

Comment:
Should the description also reflect that it is not only GitHub?

Reply:
Sure. Why not.